### ISSUE TITLE
Method POST not allowed

### ISSUE THREAD

**Issue by begin29** [intridea/grape#1077]
I got error `Method Not Allowed` when called `POST` method.

I got this error when updated grape from `0.9.0` to `0.12.0`  

```json
Response headers
Allow:OPTIONS, GET, HEAD
...
```

**Comment by dblock** [comment]
Please post a repro, obviously POST is allowed :)

**Comment by begin29** [comment]
My project contains Rails(with grape gem) + EmberJS. 

I received this error when try to login via request POST 'login_url'. 
For authenticate on server side I've used `devise`. 
This worked on `0.9.0` grape version.

**Comment by dblock** [comment]
I want to help you, but there's nothing to work with here. Maybe try to put a project up that reproduces the problem?

**Comment by dblock** [comment]
Btw, https://github.com/dblock/grape-on-rails is a working example of a recent Grape w/ Rails.

**Comment by begin29** [comment]
Actually I found problem. 
It didn't work when I've created `resource :users` and created here only '/users/:id' route and tried to made post request like '/users/sign_in'. 
`/users/sign_in` route created on routes.rb file but `grape` overrides my routes from routes.rb and returned error(Method not allowed) to any post request to `/users/some_not_matched_route`.

How can I include my routes from routes.rb file?
